.gitignore

```diff
@@ -0,0 +1,27 @@
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
```

Comment:
Can we remove this file please? none of these files or folders come up during development of this package.

Reply:
Done